test/topology/util.py

```diff
@@ -74,13 +74,14 @@ async def check_token_ring_and_group0_consistency(manager: ManagerClient) -> Non
         assert token_ring_ids == group0_ids
 
 
-async def wait_for_token_ring_and_group0_consistency(manager: ManagerClient, deadline: float) -> None:
+async def wait_for_token_ring_and_group0_consistency(manager: ManagerClient, timeout: int = 30) -> None:
```

Comment:
Even if we want to use common deadline for all servers like before, calculating it here from the timeout like this patch does, is a good idea.